gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -80,13 +80,101 @@ properties:
       - CD79A
       - CEA
       - HLA-DR
+      - Ki67
       - Mesothelin
       - NSE
       - Prostate-Specific Antigen (PSA)
       - Squamous Cell Carcinoma Antigen (SCCA)
       - Unknown
       - Not Reported
     enumDef:
+      BCL6:
+        $ref:
+          - "_terms.yaml#/bcl6/common"
+      CA-125:
+        $ref:
+          - "_terms.yaml#/ca125/common"
+      CA19-9:
+        $ref:
+          - "_terms.yaml#/ca199/common"
+      CCND1:
+        $ref:
+          - "_terms.yaml#/ccnd1/common"
+      CD10:
+        $ref:
+          - "_terms.yaml#/cd10/common"
+      CD117:
+        $ref:
+          - "_terms.yaml#/cd117/common"
+      CD138:
+        $ref:
+          - "_terms.yaml#/cd138/common"
+      CD14:
+        $ref:
+          - "_terms.yaml#/cd14/common"
+      CD15:
+        $ref:
+          - "_terms.yaml#/cd15/common"
+      CD19:
+        $ref:
+          - "_terms.yaml#/cd19/common"
+      CD20:
+        $ref:
+          - "_terms.yaml#/cd20/common"
+      CD22:
+        $ref:
+          - "_terms.yaml#/cd22/common"
+      CD23:
+        $ref:
+          - "_terms.yaml#/cd23/common"
+      CD25:
+        $ref:
+          - "_terms.yaml#/cd25/common"
+      CD3:
+        $ref:
+          - "_terms.yaml#/cd3/common"
+      CD30:
+        $ref:
+          - "_terms.yaml#/cd30/common"
+      CD33:
+        $ref:
+          - "_terms.yaml#/cd33/common"
+      CD34:
+        $ref:
+          - "_terms.yaml#/cd34/common"
+      CD45:
+        $ref:
+          - "_terms.yaml#/cd45/common"
+      CD5:
+        $ref:
+          - "_terms.yaml#/cd5/common"
+      CD56:
+        $ref:
+          - "_terms.yaml#/cd56/common"
+      CD7:
+        $ref:
+          - "_terms.yaml#/cd7/common"
+      CD79A:
+        $ref:
+          - "_terms.yaml#/cd79a/common"
+      CEA:
+        $ref:
+          - "_terms.yaml#/cea/common"
+      HLA-DR:
+        $ref:
+          - "_terms.yaml#/hladr/common"
```

Comment:
I think we need the "Ki67" enumDef here.

Reply:
@wwysoc2 Ki67 is one of the terms that doesn't exist at NCIt yet, so we have to request a new term for the next release.